client-frameworks-support/testing-utilities/dist/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@luigi-project/testing-utilities",
+  "description": "Luigi testing utilities for standalone testing of microfrontends",
+  "license": "Apache-2.0",
+  "main": "index.js",
+  "types": "index.d.ts",
```

Comment:
Now there are two package.json's 
1. testing-utilities/package.json
2. testing-utilities/**dist**/package.json

Shouldn't the **dist** folder usually be in `.gitignore` .
When this package is published, first `npm run build` will push all to **/dist** folder, and then the output build will be released from here targeting the dist folder : https://github.com/SAP/luigi/blob/44f357891a7bfccfe39d8828e51f83cfd27401ca/scripts/npmPublish.sh#L151-L154

Reply:
It's not necessarily that dist should be completeley in gitignore. E.g. imo core/public, client/public and client-support-ui5 also have an package.json which is not ignored and the public folder is equivalent to dist.

The reason why I want to avoid to copy the package.json is that script commands should not be in published package.json. In addition the devDependencies should be in the public package.json.

I don't get your point about the npmPublish completely. Maybe offline :)

Reply:
ahh true I forgot about that. I'll try to test and let you know.